src/components/PreferencesBodyApplication.vue

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+import Vue from 'vue';
+import Tabbed from '@/components/Tabbed/index.vue';
+import Tab from '@/components/Tabbed/Tab.vue';
+import PreferencesApplicationBehavior from '@/components/PreferencesApplicationBehavior.vue';
+import PreferencesApplicationEnvironment from '@/components/PreferencesApplicationEnvironment.vue';
+
+export default Vue.extend({
+  name:       'preferences-body-application',
+  components: {
+    Tabbed, Tab, PreferencesApplicationBehavior, PreferencesApplicationEnvironment
+  },
+  data() {
+    return { activeTab: 'environment' };
+  },
+  methods:    {
+    tabSelected({ tab }: { tab: any }) {
```

Comment:
```suggestion
    tabSelected({ tab }: { tab: typeof Tab }) {
```
At least, that's my best guess as to what type it should be?

Reply:
On closer inspection, it looks like something like ` tabSelected({ tab }: { tab: Vue.Component }) {` is more appropriate. I'm wondering if there's some `VueComponent` type that would fit better, but I can't find it..